packages/design-system-web/src/components/geov-entity-props-by-predicate/geov-entity-props-by-predicate.css

```diff
@@ -12,3 +12,17 @@ ion-card {
 geov-paginator {
   margin-left: auto;
 }
+
+.litteral-container {
+  overflow: clip;
+  width: 100%;
+}
+
+.litteral-expanded ion-label {
+  white-space: normal !important;
+}
+
+.text-right {
+  text-align: right !important;
```

Comment:
Never use !important. Add a little specificity in the selector and you won't need !important.

Reply:
Done